routes/api_v2_0.py

```diff
@@ -513,6 +515,9 @@ def subset_query(
         timestamp=int(time_range[0]),
         endtime=int(time_range[1]),
     ) as dataset:
+        
+        if depth is not None:
+            args['depth']=depth
```

Comment:
depth was already been added to `args` on 495 so this can be removed. A `None` value can be handled in the subset function